app/assess/routes.py

```diff
@@ -3,6 +3,10 @@
 from app.assess.data import get_funds
 from app.assess.data import get_round
 from app.assess.data import get_rounds
+from app.assess.status import get_status_COMPLETED
```

Comment:
change to all lower case, reserve all caps for constants. 😸

Reply:
Done :)